data_management/phase_tkinter_class.py

```diff
@@ -66,14 +66,16 @@ def __init__(self, next_phase, master=None, prev_phase: tk.Frame = None, test: b
 
         else:  # there is a previous phase
             self.np_img = prev_phase.np_img
-            self.story_folder = prev_phase.story_folder
-            self.filename = prev_phase.filename
+            self.os_story_folder = prev_phase.story_folder
+            self.os_filename = prev_phase.filename
 
-        self.photo_image_filename_only = path.basename(self.filename)
-        self.photo_image_folder = path.dirname(self.filename)
+        self.os_photo_image_filename_only = path.basename(self.os_filename)
+        self.os_photo_image_folder = path.dirname(self.os_filename)
 
-        text_full_path = path.join(self.story_folder,self.photo_image_filename_only.split(".")[0])
+        text_full_path = path.join(self.os_story_folder, self.os_photo_image_filename_only.split(".")[0])
```

Comment:
it could be helpful for readability to abstract this logic into some kind of function that has a descriptive name of what its doing